source/js/components/join/join.jsx

```diff
@@ -261,17 +263,21 @@ export default class JoinUs extends React.Component {
           {!this.state.apiSuccess ? `${this.props.ctaHeader}` : `Thanks!`}
         </h5>
         {!this.state.apiSuccess ? (
-          <p
+          <div
             dangerouslySetInnerHTML={{
               __html: this.props.ctaDescription
             }}
           />
         ) : (
-          <p
-            dangerouslySetInnerHTML={{
-              __html: this.props.thankYouMessage
-            }}
-          />
+          <React.Fragment>
+            <p>
+              {getText(`If you haven’t previously confirmed your opt-in to a
+              Mozilla-related email subscription you may have to do so now.`)}
```

Comment:
This doesn’t work, because the leading spaces are considered part of the key here. So that string can’t be found.
Let’s probably just use a shorter key so that it fits in one line?

Reply:
Good point! Will update!